src/System.Net.NetworkInformation/pkg/win/System.Net.NetworkInformation.pkgproj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <PackageTargetRuntime>win7</PackageTargetRuntime>
+    <PreventImplementationReference>true</PreventImplementationReference>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- bring in both the win32 and winrt implementations,
```

Comment:
cleaned up for System.Net.NetworkInformation, System.Net.NameResolution, System.Net.Primitives, and System.Net.Sockets